pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -123,6 +125,31 @@ func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEventWithHint {
 	return events
 }
 
+// TODO: kube-schduler should unregister the CSINode events once all the volume plugins has completed their CSI migration.
+func (pl *VolumeBinding) isSchedulableAfterCSINodeChange(logger klog.Logger, pod *v1.Pod, oldObj, newObj interface{}) (framework.QueueingHint, error) {
+	oldCSINode, modifiedCSINode, err := util.As[*storagev1.CSINode](oldObj, newObj)
+	if err != nil {
+		return framework.Queue, err
+	}
+
+	logger = klog.LoggerWithValues(
+		logger,
+		"Pod", klog.KObj(pod),
+		"CSINode", klog.KObj(modifiedCSINode),
+	)
+
+	oldAnn := oldCSINode.ObjectMeta.Annotations[v1.MigratedPluginsAnnotationKey]
+	modifiedAnn := modifiedCSINode.ObjectMeta.Annotations[v1.MigratedPluginsAnnotationKey]
+
+	if oldAnn != modifiedAnn {
+		logger.V(5).Info("CSINode was created or updated and this may make the Pod rejected by VolumeBinding plugin in the previous scheduling cycle schedulable")
+		return framework.Queue, nil
+	}
```

Comment:
```suggestion
	if oldCSINode.ObjectMeta.Annotations[v1.MigratedPluginsAnnotationKey] != modifiedCSINode.ObjectMeta.Annotations[v1.MigratedPluginsAnnotationKey] {
		logger.V(5).Info("CSINode's migrated plugins annotation is updated and that may make the pod schedulable")
		return framework.Queue, nil
	}
```

Reply:
@sanposhiho 
I fix this commit [6e537c5](https://github.com/kubernetes/kubernetes/pull/125097/commits/6e537c571b4918dbef5e2ece5ea41076aa5ab519)